test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -698,13 +742,16 @@ namespace integration_framework {
       std::function<void(const shared_model::proto::TransactionResponse &)>
           validation) {
     // fetch first response associated with the tx from related queue
-    TxResponseType response;
-    fetchFromQueue(responses_queues_[tx_hash.hex()],
-                   response,
-                   tx_response_waiting,
-                   "missed status");
+    boost::optional<TxResponseType> opt_response;
+    const auto it = responses_queues_.find(tx_hash.hex());
+    if (it != responses_queues_.end()) {
+      opt_response = it->second->try_pop();
+    }
+    if (not opt_response) {
+      throw std::runtime_error( "missed status");
```

Comment:
```suggestion
      throw std::runtime_error("missed status");
```
Looks like file should be clang-formatted.